visualvm/build-nb-ml.sh

```diff
@@ -1,13 +1,76 @@
-#!/bin/sh
+#!/bin/bash
+
+REV=41da26b7de7506c8
+REV_L10N=1180ea1ceb30
+ZIPNAME=nb90_visualvm_`date "+%d%m%Y"`
```

Comment:
This is a build script that prepares the right `nb90_visualvm_xyz.zip` and its `ml` variant by checking out the right sources, building them and adjusting them for purposes of VisualVM.